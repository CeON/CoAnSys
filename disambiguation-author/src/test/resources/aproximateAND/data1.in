c1	Kowalski	[EX_TITLE#{(An Empirical Study on the AND)},EX_KEYWORDS#{(2011)}]
c2	Kowalski	[EX_TITLE#{(An Empirical Study on the AND)},EX_KEYWORDS#{(2012)}]
c3	Kowalski	[EX_TITLE#{(An Empirical Study on the BAND)},EX_KEYWORDS#{(2013)}]
c4	Kowalski	[EX_TITLE#{(An Empirical Study on the BAND)},EX_KEYWORDS#{(2014)}]
c5	Kowalski	[EX_TITLE#{(An Empirical Study on the AND)},EX_KEYWORDS#{(2014)}]
