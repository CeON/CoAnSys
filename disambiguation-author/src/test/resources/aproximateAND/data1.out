({(c1,Kowalski,[EX_KEYWORDS#{(2011)},EX_TITLE#{(An Empirical Study on the AND)}]),(c2,Kowalski,[EX_KEYWORDS#{(2012)},EX_TITLE#{(An Empirical Study on the AND)}]),(c3,Kowalski,[EX_KEYWORDS#{(2013)},EX_TITLE#{(An Empirical Study on the BAND)}]),(c4,Kowalski,[EX_KEYWORDS#{(2014)},EX_TITLE#{(An Empirical Study on the BAND)}]),(c5,Kowalski,[EX_KEYWORDS#{(2014)},EX_TITLE#{(An Empirical Study on the AND)}])},{(1,0,0.0),(3,2,0.0),(4,0,0.0),(4,3,0.0)})
